helm/cloud-provider-vsphere/charts/kube-vip/values.yaml

```diff
@@ -0,0 +1,100 @@
+# Default values for kube-vip.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+image:
+  repository: ghcr.io/kube-vip/kube-vip
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "v0.5.11"
+
+config:
+  address: ""
+
+env:
+  vip_interface: "eth0"
+  vip_arp: "true"
+  lb_enable: "true"
+  lb_port: "6443"
+  vip_cidr: "32"
+  cp_enable: "false"
+  svc_enable: "true"
+  vip_leaderelection: "false"
+
+envFrom: []
+  # Specify an externally created Secret(s) or ConfigMap(s) to inject environment variables
+  # For example an externally provisioned secret could contain the password for your upstream BGP router, such as
+  #
+  # apiVersion: v1
+  # data:
+  #   bgp_peers: "<address:AS:password:multihop>"
+  # kind: Secret
+  #   name: kube-vip
+  #   namespace: kube-system
+  # type: Opaque
+  #
+#- secretKeyRef:
+#    name: kube-vip
```

Comment:
Is it really usable? I feel most (if not all) of env vars will be overwritten by those defaults from `.Values.env` anyway.

Reply:
I didn't expose any of these variables at the top chart level. These come from original chart https://github.com/kube-vip/helm-charts/blob/main/charts/kube-vip/values.yaml#L24

I tried to change minimum things in the upstream chart.